src/main/java/htsjdk/samtools/util/IOUtil.java

```diff
@@ -97,6 +96,11 @@
 
     private static int compressionLevel = Defaults.COMPRESSION_LEVEL;
 
+    /** signature of a gzip stream used by {@link isGZIPInputStream} 
+     * @see {@linkplain http://stackoverflow.com/questions/4818468/how-to-check-if-inputstream-is-gzipped}
+     */
+    public static final byte GZIP_SIGNATURE[]= {(byte) 0x1f, (byte) 0x8b};
```

Comment:
It would be better I think to use the `GZIPInputStream.GZIP_MAGIC` constant directly instead of reproducing it here?